### PR TITLE
Updated gems, fixed test

### PR DESCRIPTION
**Pull request by evgeniyvlb**
Upgrade gems, update tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -22,13 +27,13 @@ env:
...
+    - secure: "k/2zzUeePe753UyPxCqCiiIwafhO19V0U4NOo1IToZUYwsGQ8L3uzzpNU8PfJDYkPYWTZBnvQNhm3gJVgvGvsVpzawmWVUHtkX+1bL8+dpVnqBjrlnR3+Du6OsLT7PtGGWOFuZDG3XDpQumI2vLq1osDZsnYB9Y+5cDY1Siu4GMY2G/jEOW5I8khufo4ubDhsY5vEmz1T+7N7G4lmyQy3Io3Jh3Ciws1Cku97ZbBT3RB+sPMlbAy40b0zZEZr0BuSogEUzfJdEt8nPQhd2nregwSkaeN9NgFVWl7nFdmBFTxr5wf86sUHH7vEd40k4uKx8BOYdtVcFAMWK/QmSXIQwO1WCHt1Fi2MloIIzGb7FkDuIS6cVbUm1GCtEArYHhX+Eti5Me/3j1hCGVtUwzNUv4PjocJ7pwy+H0DqRRwvTf77dUVVzBKMHcXmm8MMMu/AT0iTQTfl3R4JX3WgtSnxzLUAoa48K2gXWFqi81lwSG7C0lTvPG6RPF5guY+MgC1lNXVRRV5+OfnPgStwCi9/N7JLddLvmo8S8luAK8bAPcAxxQ7mX38/pQvC1BXWq7EMSJhIP6vTLz2T5foDv8vCVAg6x7H4hnbNzCu2qBIl9o55wcbIQTC0ACaEeDOdI7hHRVTluKIfMqnWr0si9k4CK45tiktZF9/bGjKQGQxH5A=" #driver=poltergeist
```
Comment (romikoops) [inline .travis.yml]: Why do you secure not secured information here? Have you introduced additional settings?

File: scripts/run-tests.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@ then
     wget https://chromedriver.storage.googleapis.com/2.38/chromedriver_linux64.zip
     unzip chromedriver_linux64.zip
     sudo cp chromedriver /usr/local/bin/chromedriver
+    #sudo ln -s /usr/local/bin/chromedriver /usr/bin/chromedriver
```
Comment (romikoops) [inline scripts/run-tests.sh]: I would remove commented code

File: scripts/run-tests.sh [hunk 3]
```diff
@@ -13,7 +14,12 @@ then
     sh -e /etc/init.d/xvfb start &
     sleep 3
 fi
-bundle exec rake rubocop features:smoke
+if [[ "$SEXY_SETTINGS" =~ .*poltergeist.* ]]
+then
+    bundle exec rake rubocop features:smoke_except
```
Comment (romikoops) [inline scripts/run-tests.sh]: `smoke` tests are clear test group definition. Unfortunately, smoke_except is not. Moreover it introduces one more group which is not documented and will be replaced with next Howitzer upgrade procedure. What about marking some scenarios which we are not willing to be executed under Poltergeist driver with `no_poltergeist` tag. I would exclude all scenarios when marked with the tag for **ALL** group of tests when current driver is `poltergeist`

File: .travis.yml (hunk 1 above)
Comment (evgeniyvlb) [reply .travis.yml]: Yes, I've added private variables.

**Pull request merged**